dags/sentinel-2_data_transfer.py

```diff
@@ -71,100 +94,159 @@ def africa_tile_ids():
 
     return africa_tile_ids
 
-def publish_to_sns_topic(message, attribute):
+
+def publish_to_sns_topic(message):
     """
     Publish a message to a SNS topic
     param message: message body
     """
 
-    sns_hook = AwsSnsHook(aws_conn_id=dag.default_args['africa_conn_id'])
+    body = json.loads(message.body)
+    attributes = body.get("MessageAttributes", {})
+    attributes["product"] = attributes.pop("collection")
```

Comment:
Why pop it to another field?
```suggestion
    del attributes["collection"]
```

Reply:
This was to replace the old key with the new key.  I've done it your way now.